src/main/java/com/softserveinc/dokazovi/controller/PostController.java

```diff
@@ -130,4 +134,23 @@
 				.status((postDTO != null) ? HttpStatus.OK : HttpStatus.NOT_FOUND)
 				.body(postDTO);
 	}
+
+	@DeleteMapping(POST_DELETE_BY_ID)
```

Comment:
Implicit endpoint. Finally it will look:
`localhost:8080/api` + `/post/` + `/{postId}` - _unclear what will do this endpoint_

Reply:
`localhost:8080/api` + `/post/` + `/{postId}` + `/delete' - hope will look better